src/frontend/src/lib/types/token.ts

```diff
@@ -26,11 +26,16 @@ export interface TokenMetadata {
 }
 
 export interface TokenAppearance {
+	oisySymbol?: TokenOisySymbol;
 	oisyName?: TokenOisyName;
 }
 
+export interface TokenOisySymbol {
+	oisySymbol: string;
+}
+
 export interface TokenOisyName {
-	prefix: string | undefined;
+	prefix?: string | undefined;
```

Comment:
this change is expected?

Reply:
ah yes, sorry, I forgot to mention it in the description, but it is sort of unrelated, just a "since we are here..."

Do you prefer it in another PR?

Reply:
This PR is ok.

Since we are here, you didn't want to shorten it to `prefix?: string` or keeping the `| undefined` is on purpose?

Reply:
nono, you are right, it is better